src/t2iapi/context/context_requests.proto

```diff
@@ -70,3 +70,15 @@ Request to associate a specific patient type.
 message AssociatePatientRequest {
     PatientType patient_type = 1;  // Patient type to be associated
 }
+
+/*
+Request to create or adapt a context state for the given descriptor handle with the starting @ContextAssociation value
+given as "start_association", to perform a transition to the final @ContextAssociation value, given as
+"end_association", happens during the remote procedure call.
```

Comment:
I don't understand the "happens during the remote procedure call.", that doesn't fit into the sentence. 
> to perform a transition to the final @ContextAssociation value [...] happens during the remote procedure call.

Reply:
I removed it.